Content.Server/Chat/Systems/ChatSystem.cs

```diff
@@ -60,20 +60,26 @@ public sealed partial class ChatSystem : SharedChatSystem
     private bool _loocEnabled = true;
     private bool _deadLoocEnabled = false;
     private readonly bool _adminLoocEnabled = true;
+    private string _motd = "";
+    public string MOTD => _motd;
```

Comment:
Why is this tacked onto the (already massive) ChatSystem.
Give it it's own system.